src/Compilers/Core/Portable/Operations/Queries/OrderKind.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Kind of ordering for an <see cref="IOrderingExpression"/>
+    /// </summary>
+    public enum OrderKind
+    {
+        /// <summary>
+        /// Represents no ordering for an <see cref="IOrderingExpression"/>.
+        /// </summary>
+        None = 0x0,
```

Comment:
is this possible?  seems unlikely.  if not, i would recommend removing enum and just having a bool "IsDescending"